.travis.yml

```diff
@@ -184,10 +216,10 @@ script:
   # Each script we want to run need to go in its own section and the program
   # you want to fail travis needs to be the last thing called.
   - |
-    # The number of processes is hardcoded (-n2), because using too many
+    # The number of processes is hardcoded (-n4), because using too many
```

Comment:
we get 4 usable cores on travis now?

Reply:
In Arm64, we can use more than two cores, please refer "https://travis-ci.community/t/nproc-reports-32-cores-on-arm64/5851/2". In amd64  it is picking 4 workers for amd64 as well, but it maybe running on 2 cores only.
but it("-n4") has significant difference in time for Arm64 platform.

Reply:
The core count wasn't the problem, it was opening too many Inkscape and Ghostscript processes, as noted in the comment below. Though now we pipe everything into a single Inkscape instance (and Ghostscript?) per process, so maybe that's not an issue.